hack/test-cmd.sh

```diff
@@ -78,9 +84,15 @@ set -e
 
 function find_tests {
   find "${OS_ROOT}/test/cmd" -name '*.sh' -not -wholename '*images_tests.sh' | grep -E "${1}" | sort -u
```

Comment:
If no tests are selected, the user made a mistake in 99% of cases. Handling that with a nice `os::log::error` and a failure seems fine to me.